origin.html

```diff
@@ -12,26 +12,85 @@
-        <h2>Let's get in touch...!</h2>
-        <ul class="socialmedia">
-            <li><a href=""><img src="./images/github.svg" alt="githublogo"></a></li>
-            <li><a href=""><img src="./images/Linkedinicon.svg" alt=""></a></li>
-            <li><a href=""><img src="./images/angellist.svg" alt="angellist"></a></li>
-            <li><a href=""><img src="./images/twitter.svg" alt="twitterlogo"></a></li>
-            <li><a href=""><img src="./images/medium.svg" alt="mediumlogo"></a></li>
-        </ul>
+        <div class="firstsection">        
+            <h1>Hey there, I'm Andre</h1>
+            <h2>I'm a software developer with experience in finance and logistic processes.</h2>
+            <p class="firstparagraph">I can build a website meeting the expected requirements, pluss I can add my actual knowledge of how the process need to work on certain areas like logistic, account, finance and billing and all of this focus to the good experience and satisfaction ot the customer.</p>
+            <ul class="socialmedia">
+                <li><a href=""><img src="./images/github.svg" alt="githublogo"></a></li>
+                <li><a href=""><img src="./images/Linkedinicon.svg" alt=""></a></li>
+                <li><a href=""><img src="./images/angellist.svg" alt="angellist"></a></li>
+                <li><a href=""><img src="./images/twitter.svg" alt="twitterlogo"></a></li>
+                <li><a href=""><img src="./images/medium.svg" alt="mediumlogo"></a></li>
+            </ul>
+        </div>
+        <div class="storysection">
+            <h2 class="storytitle">My Recent Work</h2>
+            <ul class="storyimages">
+                <li><img class="underline" src="./images/story/Vector.png" alt=""></li>
+                <li><a href=""><img class="secondmain" src="./images/SecondPage/ImgPlaceholder.png" alt=""></a></li>
+            </ul>
+            <h3 class="storysubtitle">Multi-Post Stories</h3>
+            <p class="storyparragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="storyimages2">
+                <li class="img1"><a href=""><img class="skill-image-tag1" src="./images/SecondPage/Tag.png" alt="css"></a></li>
+                <li class="img2"><a href=""><img class="skill-image-tag1" src="./images/SecondPage/Tag1.png" alt=""></a></li>
+                <li class="img3"><a href=""><img class="skill-image-tag1" src="./images/SecondPage/Tag2.png" alt=""></a></li>
+                <li class="img4"><a href=""><img class="skill-image-tag1" src="./images/SecondPage/Tag3.png" alt=""></a></li>
+            </ul>
+            <p><button class="second">See Project</button></p>
```

Comment:
[OPTIONAL] Please make sure your buttons are identical in design with the ones shown in [chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). As a future web developer, you will have to build UI based on the design specified in the technical requirements, let's get used to it from the scratch :relaxed: 

| The design you implemented  | Original design in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4) |
| ------------- | ------------- |
| ![Screenshot from 2022-02-12 19-37-46](https://user-images.githubusercontent.com/67608876/153715644-411a1a5f-2602-495a-b6e4-2a43fddd2896.png) | ![Screenshot from 2022-02-12 19-39-19](https://user-images.githubusercontent.com/67608876/153715714-9900578a-ce72-409e-b34c-f7fe15b74d78.png) |

_note: this comment refers to all `buttons` in your project._